src/proc/hdr-merge.cpp

```diff
@@ -258,6 +258,13 @@ namespace librealsense
         // checking frame counter of first depth and ir are the same
         if (use_ir)
         {
+            // on devices that does not support meta data on IR frames, do not use IR for hdr merging
```

Comment:
does -> do

Reply:
Would be nice if it explained which devices did or did not support metadata on IR frames...

Reply:
Added

Reply:
Need to add to development too, no back merge is planned right?

Reply:
Right